### DISCUSSION TITLE
How do you remove a portion of the line shader that is overlapping of the separate obeject

### DISCUSSION THREAD

**Discussion by dundundunnn** [bnpr/Malt#512] [Q&A]
I'm trying to remove this portion of the line shader that is overlapping between separate objects but want to keep the rest of the line shader
![blender_AysYQ9Ymvk](https://github.com/bnpr/Malt/assets/48817977/c3c87764-8d9a-406f-846c-8491eeb20764)

**Comment by pragma37** [comment]
Here's an explanation of the Line Width node parameters https://github.com/bnpr/Malt/discussions/274#discussioncomment-2398023.
And here's a very similar question https://github.com/bnpr/Malt/discussions/474#discussioncomment-4747333.
